eva/executor/lateral_join_executor.py

```diff
@@ -46,3 +46,6 @@ def exec(self, *args, **kwargs) -> Iterator[Batch]:
                 result_batch = apply_project(result_batch, self.join_project)
                 if not result_batch.empty():
                     yield result_batch
+
+    def __call__(self, *args, **kwargs) -> Generator[Batch, None, None]:
```

Comment:
@xzdandy @kaushikravichandran Do we need to do it for all the executors?

Reply:
If yes, how about adding it to the `AbstractExecutor`?

Reply:
Yes. That's good point. However, not all `exec` has implemented the `*arg` and `**kwargs`